requirements-transitive.txt

```diff
@@ -11,6 +11,7 @@ aspy.refactor-imports==2.1.1
 attrs==20.3.0
 Babel==2.9.1
 beautifulsoup4==4.9.3
+boto3==1.21.20
```

Comment:
boto3 should be a direct dependency, not a transitive one? you are importing it directly.

Reply:
I'm not really sure what the difference is. I noticed redis was imported in requirements-transitive.txt for example. Does it matter? For now I moved it to requirements.txt, but it seems to be a much smaller file. Looks like it's used for style/python stuff rather than libraries?